### DISCUSSION TITLE
Consulo for WindowWindows

### DISCUSSION THREAD

**Discussion by NwosuTy** [consulo/consulo#488] [Q&A]
please i use a windows 10  os id like to know how to configure  consulo for just  c# development  not unity

**Comment by VISTALL** [comment]
Hello. What framework you will use ? Mono/.NET Framework/.NET Core ?

**Reply by NwosuTy** [reply]
.net framework  thank you

**Comment by VISTALL** [comment]
Check Settings -> Platform & Plugins:

* .NET Framework
* MSBuild
* C# 
If not installed - click From Repository, and instsall them

**Reply by NwosuTy** [reply]
Thank You

Would i need to configure anything after installation

**Reply by VISTALL** [reply]
Then try "Open or Import Project" from welcome screen, select .sln file inside your project

**Reply by NwosuTy** [reply]
thank you ive got one last problem but this time for  consulo with unity 

![Lets Fight - NewBehaviourScript cs  Assembly-CSharp  - Consulo 2021 10 15-Nov-21 6_38_37 AM](https://user-images.githubusercontent.com/94308549/141728526-edc5ff0a-970c-4c53-b323-a2e738ec350e.png)

**Reply by NwosuTy** [reply]
i keep trying to select from path and it keeps saying "The selected directory is not a valid home for Unity3D"

**Reply by VISTALL** [reply]
What path you want to select?
